public/views/partials/footer.html

```diff
@@ -0,0 +1,17 @@
+<footer id="footer">
+	<div class="scroll-to-top">
+		<a href="" onclick="scrollToTop();">Top</a>
+	</div>
+	<div class="footer-container">
+		<ul class="footer-link-collection">
+            <li><a class="footer-link" href="/contact">Contact Us</a></li>
+            <li><a class="footer-link" href="/media">Social Media</a></li>
+            <li><a class="footer-link" href="/sponsors">Sponsors</a></li>
```

Comment:
I was thinking that we could have this expand into a menu (or just have a separate series of icons) where we have facebook, twitter, instagram and the such. I feel like only writing facebook or just putting a solitary facebook icon looks weird.